pom.xml

```diff
@@ -63,7 +63,7 @@
     </pluginRepositories>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.479</jenkins.version>
+        <jenkins.version>2.480</jenkins.version>
```

Comment:
Given this just adds a test, I am not sure if we actually want to merge this as-is, or would prefer to wait for JENKINS-73824 to possibly get backported to 24.79.x.

Reply:
Oh OK, it is an `lts-candidate`.